### ISSUE TITLE
Not an issue but a question.

### ISSUE THREAD

**Issue by bigtiny** [jrnelson90/tinderboxpedal#7]
How did you get the bluetooth codes for the different buttons and is there more code that could be used for different functions?

**Comment by jrnelson90** [comment]
Yes, I'm working on code for more functionality, but at the moment I don't have a build stable enough yet to feel comfortable releasing it to others yet. 

As for how, I use Android Debugginh and Wireshark to see what BT commands (packets) are being sent when you edit the tone in the app. Then, like a universal remote, we can use that in the project to control the amp.
